plugins/wdc/wdc-nvme.c

```diff
@@ -775,7 +775,36 @@ struct __attribute__((__packed__)) wdc_nand_stats {
 	__le64		nand_rec_trigger_event;
 	__le64		e2e_error_counter;
 	__le64		successful_ns_resize_event;
-	__u8		rsvd[444];
+	__u8		rsvd[442];
+	__u16       log_page_version;
+};
+
+struct __attribute__((__packed__)) wdc_nand_stats_V3 {
+	__u8		nand_write_tlc[16];
+	__u8		nand_write_slc[16];
+	__le64		bad_nand_block_count;
+	__le64		xor_recovery_count;
+	__le64		uecc_read_error_count;
+	__u8		ssd_correction_counts[16];
+	__u8        percent_life_used;
```

Comment:
Please use tabs consistently. This here is a mix of tabs and spaces (this project's coding style prefers tabs; it's the linux way).